Assets/lang/en.json

```diff
@@ -8,6 +8,10 @@
         "logout": "logout"
     },
     "mca_ayim": {
+		"banner":{
+			"merch": "OFFICIAL MERCHANDISE STORE",
+			"corsace": "MAIN EVENT HUB"
+		},
```

Comment:
Fix spacing for the new additions in this whole file

Reply:
`"banner":{` nmeeds a space between : {

Reply:
added space